### ISSUE TITLE
ShiftNegative substitution doesn't keep the non-linear node number of activation bits, after it was changed with the model_editor

### ISSUE THREAD

**Issue by elad-c** [sony/model_optimization#145]
to reproduce:
1. run the MCT with a network with swish activations (e.g. "mobile_vit")
2. change the swish nodes activation bits to something other that 8 with the model_editor (e.g. 7)
3. after the substitution, verify the swish node activation bits is 16, and the post_add activation bits is 7

setting the model_editor:
`
model_editor_list = []
model_editor_list.append(EditRule(filter=node_filters.NodeTypeFilter(tf.keras.layers.Softmax),
                                                        action=edit_actions.ChangeCandidatesActivationQuantConfigAttr(activation_n_bits=16)))
`
then pass the model_editor_list to the mct.keras_post_training_quantization in the network_editor argument
